analysis/Scripts/Efficiencies/efficiencies.py

```diff
@@ -0,0 +1,191 @@
+from ROOT import TChain, TCanvas, TH1, TFile
+import ROOT, os, Imports, sys, getopt
+from Imports import TUPLE_PATH, RAW_TUPLE_PATH, MC_jobs_Dict
+import pprint
+
+ybins = Imports.getYbins()
+ptbins = Imports.getPTbins()
+
+def main(argv):
+
+	ROOT.gROOT.SetBatch(True) #STOP SHOWING THE GRAPH FOR ROOT
+
+	try:
+		opts, args = getopt.getopt(argv,"hstp")
+	except getopt.GetoptError:
+		print("The arguments are wrong")
+		sys.exit(2)
+
+	options = []
+	arguments = []
+
+	for opt,arg in opts:
+		options.append(opt)
+		arguments.append(arg)
+
+	if not options:
+		options = ["-s","-t","-p"]
+
+	if "-h" in options:
+		print(textwrap.dedent("""\
+
+			Welcome to the efficiencies.py script.
+
+			The parameters are
+				-h : help
+				-s : Selection
+				-t : Trigger
+				-p : PID
+
+			Running with no parameter will output all the data at once.
```

Comment:
all the efficiencies?

Reply:
No parameters will just run the code first for the selection, output the values etc, then do the same for trigger and then for PID (but that last one is not yet implemented). I don't know if this answeres you question?

Reply:
Sure, then I would change data for efficiencies (just to be explicit)